qt/TorrentDelegateMin.cc

```diff
@@ -266,7 +266,7 @@ void TorrentDelegateMin::drawTorrent(QPainter* painter, QStyleOptionViewItem con
     }
 
     progress_bar_style_.state = progress_bar_state;
-    progress_bar_style_.text = QStringLiteral("%1%").arg(static_cast<int>(tr_truncd(100.0 * tor.percentDone(), 0)));
+    progress_bar_style_.text = QStringLiteral("%1%").arg(static_cast<int>(tr_truncd(100.0 * tor.activityPercentDone(), 0)));
```

Comment:
Need to do the same in qt/TorrentDelegate.cc as well?

Reply:
all the progress related computations in TorrentDelegate.cc are separate for downloading and seeding (e.g. [here](https://github.com/transmission/transmission/blob/main/qt/TorrentDelegate.cc#L161)) so i dont think i need to